pkg/tasks/tasks.go

```diff
@@ -90,6 +92,14 @@ func NewTaskGroup(tasks []*TaskSpec) *TaskGroup {
 	}
 }
 
+func (tge TaskGroupErrors) Error() string {
```

Comment:
(nit) can you move it after the `TaskGroupErrors` definition?

Reply:
Done